### PR TITLE
splitting out of the getting started read me

### PR DESCRIPTION
**Pull request by danielryannimble**
## Changes

the getting started readme is too be. it now has a run tests locally, and application insights files.

## Screenshots of UI changes

not required

## Checklist

no functional changes, just documentation

### CODE REVIEW

**Comment by nwarms** [comment]
I think supercharging your dev environment should be split out as well

File: docs/application-insights.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- [Application Insights](#application-insights)
```
Comment (dynamictulip) [inline docs/application-insights.md:1]: I don't think that this link is needed anymore in this file as there's only one section in it

File: docs/application-insights.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Application Insights
```
Comment (dynamictulip) [inline docs/application-insights.md:3]: Every markdown file should have a heading line, this would a good candidate for that heading

File: docs/run-tests-locally.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+```
+3. run the tests
+
+```bash
+cd tests/playwright
+
+npx playwright test --project=zap-tests --trace=on
+```
+
+## Supercharge your dev environment
```
Comment (dynamictulip) [inline docs/run-tests-locally.md:122]: Supercharge your dev environment would be better as its own file

File: docs/run-tests-locally.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+- [Supercharge your dev environment](#supercharge-your-dev-environment)
+  - [Set up continuous testing](#set-up-continuous-testing)
+  - [Analyse test coverage](#analyse-test-coverage)
+  - [Configure linting and code cleanup](#configure-linting-and-code-cleanup)
+
+
+
+
+
+## Run tests locally
```
Comment (dynamictulip) [inline docs/run-tests-locally.md:15]: I think this should be moved to the top of the file and become a lvl 1 heading

File: docs/run-tests-locally.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
+- [Run tests locally](#run-tests-locally)
+    - [Unit tests](#unit-tests)
+    - [Accessibility and UI tests](#accessibility-and-ui-tests)
+    - [Integration and Deployment tests](#integration-and-deployment-tests)
+    - [Owasp Zap security tests](#integration-and-deployment-tests)
+- [Supercharge your dev environment](#supercharge-your-dev-environment)
+  - [Set up continuous testing](#set-up-continuous-testing)
+  - [Analyse test coverage](#analyse-test-coverage)
+  - [Configure linting and code cleanup](#configure-linting-and-code-cleanup)
+
```
Comment (dynamictulip) [inline docs/run-tests-locally.md:10]: There's a lot of unnecessary whitespace here

**Review by nwarms — DISMISSED** [review]

File: docs/getting-started.md [hunk 6]
```diff
@@ -82,194 +73,3 @@ Once you are done, ensure that you stop the container(s)!
...
-```bash
-dotnet user-secrets set "TestOverride:PlaywrightTestSecret" "TestSuperSecret"
-```
-
-This will add an authorisation header to each playwright request which should match the auth bypass secret in the playwright .env file.
-
-For more information on running and debugging Playwright tests it is worth familiarising yourself with the Playwright docs on [debugging](https://playwright.dev/docs/debug) and [command line flags](https://playwright.dev/docs/test-cli).
-
-> **Warning**
-> There can be some issues with running Docker on Windows. [See this to fix it](.\docker-issues-on-windows.md)
```
Comment (nwarms) [inline docs/getting-started.md:149]: Can you make sure this warning is still included in the test doc?

File: docs/supercharge-your-dev-environment.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+# Supercharge your dev environment
+
+Use this documentation to help supercharge your dev environment. We recommend using Rider or Visual Studio with ReSharper.
+
+- [Supercharge your dev environment](#supercharge-your-dev-environment)
+    - [Set up continuous testing](#set-up-continuous-testing)
+    - [Analyse test coverage](#analyse-test-coverage)
+    - [Configure linting and code cleanup](#configure-linting-and-code-cleanup)
+
+### Set up continuous testing
```
Comment (dynamictulip) [inline docs/supercharge-your-dev-environment.md:10]: Please can we review all the heading levels on all the new files (as we've essentially promoted all the text paragraphs) - for example, this heading should now be `##`

File: docs/run-tests-locally.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+npx playwright test --trace=on # get a time machine attached to each test result in the report
+
+# remove docker image when done
+npm run docker:stop
+```
+
+For more information on running and debugging Playwright tests it is worth familiarising yourself with the Playwright docs on [debugging](https://playwright.dev/docs/debug) and [command line flags](https://playwright.dev/docs/test-cli).
+
+**Warning**
+There can be some issues with running Docker on Windows. [See this to fix it](.\docker-issues-on-windows.md)
```
Comment (nwarms) [inline docs/run-tests-locally.md:64]:
```suggestion
> **Warning**
> There can be some issues with running Docker on Windows. [See this to fix it](.\docker-issues-on-windows.md)
```
The original was in a quote to use github's linting system

File: docs/run-tests-locally.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
+## Run tests locally
+
+Use this documentation to run tests locally.
+
+- [Run tests locally](#run-tests-locally)
```
Comment (dynamictulip) [inline docs/run-tests-locally.md:5]: We don't need this top link as it refers to the entire file

File: docs/supercharge-your-dev-environment.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
+# Supercharge your dev environment
+
+Use this documentation to help supercharge your dev environment. We recommend using Rider or Visual Studio with ReSharper.
+
+- [Supercharge your dev environment](#supercharge-your-dev-environment)
```
Comment (dynamictulip) [inline docs/supercharge-your-dev-environment.md:5]: We don't need this top link as it refers to the entire file

**Review by nwarms — APPROVED** [review]
